extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -309,9 +338,69 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
   }
 
   /** @override */
+  buildCallback() {
+    super.buildCallback();
+
+    if (this.isResponsive()) {
+      // Attempt to resize to the correct height.
+      const viewport = this.getViewport();
+      this.responsiveSizeChangePromise_ =
+          this.attemptChangeSize(
```

Comment:
Don't think so -- there's changeHeight and attemptChangeSize, but not both together.